templates/zero-httpd.conf.erb

```diff
@@ -11,13 +11,16 @@ Alias /cvmfs/<%= @repo %> <%= @repo_store %>/<%= @repo %>
     EnableMMAP Off
     EnableSendFile Off
 
-    AddType application/x-cvmfs .cvmfspublished .cvmfswhitelist
+    <FilesMatch "^\.cvmfs">
+       ForceType application/x-cvmfs
+    </FilesMatch>
+
 
     Header unset Last-Modified
     FileETag None
 
     ExpiresActive On
     ExpiresDefault "access plus 3 days"
-    ExpiresByType text/html "access plus 5 minutes"
     ExpiresByType application/x-cvmfs "access plus 2 minutes"
+    ExpiresByType application/json    "access plus 2 minutes"
```

Comment:
`application/json` is not known by SL5, hence we added  `AddType application/json .json` to the [upstream configuration](https://github.com/cvmfs/cvmfs/blob/b4f4297e54f1b65c31a79f78ee52a719ca466c65/cvmfs/cvmfs_server#L2018) just before the `<Directory ...>` block.